index.js

```diff
@@ -694,7 +700,7 @@ module.exports = function(THREE) {
         this.domElement.addEventListener( 'contextmenu', function ( event ) { event.preventDefault(); }, false );
         this.domElement.addEventListener( 'mousedown', onMouseDown, false );
         this.domElement.addEventListener( 'mousewheel', onMouseWheel, false );
-        this.domElement.addEventListener( 'DOMMouseScroll', onMouseWheel, false ); // firefox
+        this.domElement.addEventListener( 'MozMousePixelScroll', onMouseWheel, false ); // firefox
 
```

Comment:
Argh this is from my previous patch request ... let me try to do the git magic (rebase ?) to make this go away.